e2e/utils/get-env-info.ts

```diff
@@ -23,10 +23,11 @@ export function getPublishedVersion(): string {
 }
 
 export function detectPackageManager(dir: string = ''): PackageManager {
-  return existsSync(join(dir, 'bun.lockb'))
-    ? 'bun'
-    : existsSync(join(dir, 'yarn.lock'))
+  return existsSync(join(dir, 'yarn.lock'))
     ? 'yarn'
```

Comment:
made yarn the first lockfile just to make the bun/pnpm indentation identical for multiple lockfiles

Reply:
> e bun/pnpm indentation identical for multiple lockfiles

Doesnt bun current support to yarn lock still, This is why bun came frist

Reply:
right, great point, will reverse